src/Fields/state.tsx

```diff
@@ -54,9 +54,7 @@ export default props => {
                 <div className="popover-top">
                   <div className="popover-header flex-center flex-row">
                     <div className="flex-grow">
-                      <Label className="font-weight-semibold body-m">
-                        State
-                      </Label>
+                      <div className="font-weight-semibold body-m">State</div>

```

Comment:
Let's keep the Label component

Reply:
then the Label component needs to change to not specify body-s inside it

Reply:
and all instantiations of Label will have to specify their font size